test/test_cli.py

```diff
@@ -16,6 +16,21 @@ def mocked__projects(project_id, **_):
         return [{'id': 'text_project', 'inputType': 'FRAME'}]
 
 
+def test_create_project(mocker):
+    mocker.patch("kili.client.Kili.__init__", return_value=None)
+    mocker.patch("kili.client.Kili.create_project")
+    runner = CliRunner()
+    runner.invoke(create_project,
+                  ['--interface',
+                   "test/fixtures/image_interface.json",
+                   '--title',
+                   'Test project',
+                   '--description',
+                   'desciption',
+                   '--input-type',
+                   'IMAGE'])
+
```

Comment:
you can use https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once on the create_project method to check that the backend is called.